src/main/resources/application-dev.properties

```diff
@@ -3,4 +3,6 @@ javastart.fullDomainAddress=http://localhost:8080
 javastart.googleCalendarSyncEnabled=false
 spring.datasource.username=javastart
 spring.datasource.password=javastart123
-spring.datasource.url=jdbc:h2:mem:testdb
\ No newline at end of file
+spring.datasource.url=jdbc:h2:mem:testdb
+
+javastart.agreementsDir=files/agreements
```

Comment:
To naprawia sytuację wyłącznie w profilu deweloperskim, przenieś tę zmianę do pliku application.properties, albo ustaw inną ścieżkę dla profilu produkcyjnego

Reply:
tu kilka razy zmienialam wersje i ostatecznie zostawilam ta zla opcje :)